enclave-tls/src/api/enclave_tls_cleanup.c

```diff
@@ -19,9 +20,9 @@ enclave_tls_err_t enclave_tls_cleanup(enclave_tls_handle handle)
 	ETLS_DEBUG("handle %p\n", ctx);
 
 	if (!handle || !handle->tls_wrapper || !handle->tls_wrapper->opts ||
-	    !handle->tls_wrapper->opts->cleanup || !handle->attester || !handle->attester->opts ||
-	    !handle->attester->opts->cleanup || !handle->verifier || !handle->verifier->opts ||
-	    !handle->verifier->opts->cleanup)
+			!handle->tls_wrapper->opts->cleanup || !handle->attester || !handle->attester->opts ||
+			!handle->attester->opts->cleanup || !handle->verifier || !handle->verifier->opts ||
+			!handle->verifier->opts->cleanup)
```

Comment:
Please use clang-format to get rid of the codes. Refer to https://github.com/alibaba/inclavare-containers/blob/master/CONTRIBUTING.md#format-c-codes. I will continue reviewing after re-formating the codes.

Reply:
ok. Sorry for this mistake. Have correted it.